### ISSUE TITLE
New release (v0.17)

### ISSUE THREAD

**Issue by jorsn** [astroidmail/astroid#750]
I think, we should do a new release (v0.17), with the recent severe bugs and build errors fixed.
The following issues should be closed before that:

- [x] must: #744
- [x] if possible: #727 (probably fixed by #753)
- [ ] seems easy: #714 (probably fixed by #757)
- [ ] almost done: #756 (probably fixed by #757)
- [ ] low priority (to have a new release which works in usual situations): #695

not considered any more:
- [ ] ~if possible: #737~ (maybe not an issue any more)

Feel free to propose other issues.

**Comment by gauteh** [comment]
Sounds great 👍

**Comment by jorsn** [comment]
@ibuclaw @oreo639 @mklink Are any additional issues important to you for this release?

**Comment by ibuclaw** [comment]
> @ibuclaw @oreo639 @mklink Are any additional issues important to you for this release?

The only blocking issues my end were the compatibility with latest GTK since I updated by system back in April.

I think I've encountered #695 or something like what it describes before. Infrequently it can occur after polling that some threads appear twice in the thread list view. Opening such a thread crashes the program. Start astroid again and the duplicate thread is gone.

Hasn't occurred in quite some time though. Could be notmuch 0.35 vs 0.38.

**Comment by noideawhatwhat** [comment]
This one still affects Arch linux 0.16.0

https://github.com/astroidmail/astroid/issues/726

**Comment by mklinik** [comment]
Just came here on a whim, didn't see your at, I guess you meant me?

I just updated to the current master branch. Attachments work fine, but I'm seeing a long HTML email cut off and not being displayed completely, but only after I load remote images. Not a blocker though. I'll open a separate issue.

**Comment by LordGrimmauld** [comment]
Over on NixOS we are working towards deprecating `libsoup_2_4` completely, tracked in https://github.com/NixOS/nixpkgs/issues/360897.
Astroid is currently one of the few remaining packages building with the outdated (vulnerable) libsoup 2.x, until we can bump the webkitgtk dependency.

PR https://github.com/astroidmail/astroid/pull/748 added support for newer webkitgtk, but an actually tagged release would be appreciated.

**Comment by gauteh** [comment]
Released v0.17 now!
